static/styles/message_row.css

```diff
@@ -45,10 +48,16 @@ $sender_name_distance_below_flex_center: 3px;
 
             @media (width < $sm_min) {
                 padding: 0 4px;
-
                 /* This is intended to target the first message_controls child
                    when there are 3 displayed. 4 = 3 + hidden message_failed element. */
-                .message_control_button:nth-last-child(4) {
+                .message_control_button:nth-child(-n + 1) {
+                    display: none;
+                }
+            }
+            @media (width < $lg_min) {
+                /* This is intended to target the second message_controls child
+                   when there are 4 displayed. 5 = 4 + hidden message_failed element. */
+                .message_control_button:nth-child(2) {
                     display: none;
```

Comment:
This `nth-child` logic feels very hairy; how can I convince myself that it's correct.

Reply:
I referred to this document [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) while implementing this logic and thoroughly tested it under all circumstances to check if it works or not. 
Whenever the screen width is greater than lg_min (991px), there are four buttons ['emoji', 'quote and reply', 'three-dot', 'star']. The `nth-child` selector gives us an element from the front, and since `quote and reply` is the second option, that's why `nth-child(2)` is used.

Reply:
I used the `nth-child` approach since it takes the element from the start, and the respond_button is the second element to be removed. However, I found that there are some cases where this `nth-child` approach does not work. Therefore, I have replaced it and am now using just the classname to hide the button when the screen width is below 1080px.